Магия фуллеренов

LeadOnTaste opened the pull request:
Добавлены новые вещи для R-UST. Теперь R-UST куда полезнее, но требует много энергии и конские температуры для новой реакции. It's not shutting down, как говорится.:
- Наноуглеродное волокно (nanocarbon fiber) - имеет свойства пластика на стероидах;
- Реструктурированное наноуглеродное волокно - пласталь на стероидах, создаётся в R-UST;
- Наноуглеродные стержни для R-UST - создание очевидно;
- Реакция реструктуризации - позволяет из первого вида наноуглеродного волокна при очень высоких температурах получить второй;

LeadOnTaste committed:
Nanocarbon Fusion Recipe

LeadOnTaste committed:
R-UST Carbon Fuel Assembly

Карбоновые стержни для токамака

LeadOnTaste committed:
Update fuel_assembly.dm

LeadOnTaste committed:
Carbon Fiber Sheets

Добавляет углепластиковое волокно, прочнее пластика в полтора раза

LeadOnTaste committed:
Nanocarbon Fiber

Наноуглеродное волокно как материал. Оно того стоит.

LeadOnTaste committed:
Nanocarbon Fiber Explosion Protection

Наноуглеродное волокно теперь защищает от взрывов. Слава композитам!

LeadOnTaste committed:
Update materials_metal.dm

LeadOnTaste committed:
Update materials_metal.dm

LeadOnTaste committed:
Restructurized Nanocarbon Fiber

Тот самый материал из R-UST

LeadOnTaste committed:
Update fusion_reactions.dm

LeadOnTaste committed:
Update fusion_reactions.dm

LeadOnTaste committed:
Nanocarbon Fiber reagent

На всякий случай небольшой костыль для реакций

LeadOnTaste committed:
Update materials_metal.dm

LeadOnTaste committed:
Update fusion_reactions.dm

LeadOnTaste committed:
Update materials_metal.dm

HaCsO commented on code/modules/materials/definitions/materials_metal.dm:
```diff
@@ -403,3 +403,54 @@
+	burn_armor = 20
+	explosion_resistance = 45
+	value = 1000
+	ignition_point = 45000
```
![изображение](https://user-images.githubusercontent.com/48949815/147829745-a919ff5e-1006-4776-ab78-529d2e7b4a03.png)
скока

LeadOnTaste replied:
Равноценно синтезу водорода металлического

LeadOnTaste replied:
А если кратко то примерно 10 кПа на лист карбона, но проблема в том, чтобы на расчётные температуру и энергию вывести токамак

HaCsO commented on code/modules/materials/definitions/materials_metal.dm:
```diff
@@ -403,3 +403,54 @@
+	ignition_point = 45000
+	melting_point = 30000
+	alloy_product = FALSE
+	chem_products = list(
```
разве не указывается сколько единиц в единице? 50 карбона в 1 пластинке сильно...

LeadOnTaste replied:
Это при размалывании будет 50 херни и 10 форона с листа, а оно таки не реверсится, при размалывании надо будет новый лист реструктурированный делать, а для получения обычного нанокарбона нужно три листа угля сплавить

Evie7056 commented:
@LeadOnTaste Какие вообще в принципе применения у этих материалов? Помимо строительства? Строительство из этого вообще проверялось?

LeadOnTaste commented:
> @LeadOnTaste Какие вообще в принципе применения у этих материалов? Помимо строительства? Строительство из этого вообще проверялось?

Как я буду тестить когда локалка не стоит? И нет, рецепты я не завёз ещё вроде. Кроме строительства на мехов вполне сойдёт.

Evie7056 commented:
Тогда в драфт, пока у этого нет какого-либо функционала. И никто не будет проводить тесты на локалке за тебя, даже если это поставиться на сервер...

LeadOnTaste commented:
> Тогда в драфт, пока у этого нет какого-либо функционала. И никто не будет проводить тесты на локалке за тебя, даже если это поставиться на сервер...

Ну не могу я блять запускать локалку, чтоб на ней педали работали у меня, а рецепты тут у многих вещей одни, и по идее оно будет наследовать рецепты той же стали.

HaCsO commented:
> > Тогда в драфт, пока у этого нет какого-либо функционала. И никто не будет проводить тесты на локалке за тебя, даже если это поставиться на сервер...
> 
> Ну не могу я блять запускать локалку, чтоб на ней педали работали у меня, а рецепты тут у многих вещей одни, и по идее оно будет наследовать рецепты той же стали.

Не неможешь, а не хочешь.

LeadOnTaste commented:
> > > Тогда в драфт, пока у этого нет какого-либо функционала. И никто не будет проводить тесты на локалке за тебя, даже если это поставиться на сервер...
> > 
> > 
> > Ну не могу я блять запускать локалку, чтоб на ней педали работали у меня, а рецепты тут у многих вещей одни, и по идее оно будет наследовать рецепты той же стали.
> 
> Не неможешь, а не хочешь.

Именно, блять, что не могу, ибо у меня не вытягивает окно консоли с этими тремя кнопочками для запуска, захода и отключения 8 ГБ оперативки, а заход через открытие файла из меню бульона не даёт мне педали.

Evie7056 commented:
![image](https://user-images.githubusercontent.com/71145590/148242174-148ae790-9179-4685-910b-9cf0a0930b78.png)

LeadOnTaste commented:
> ![image](https://user-images.githubusercontent.com/71145590/148242174-148ae790-9179-4685-910b-9cf0a0930b78.png)

Я ПЫТАЛСЯ, НИХУЯ

LeadOnTaste committed:
Костылёк 1

По идее предотвращает триторку адгерантов и ГБС.

LeadOnTaste committed:
Костыль 1.1

Подправил переменную

LeadOnTaste committed:
Реворк спуска ГБС

Теперь в сообщении о спуске ГБС с расположенного выше Z-уровня отсутствуют крылья - они просто сползают, как и должны по лору, так как только у нимф и субимаго имеются крылья.
